ComponentKit/DataSources/CKCollectionViewTransactionalDataSource.mm

```diff
@@ -65,9 +65,13 @@ - (void)applyChangeset:(CKTransactionalComponentDataSourceChangeset *)changeset
                               userInfo:userInfo];
 }
 
-static void applyChangesToCollectionView(CKTransactionalComponentDataSourceAppliedChanges *changes, UICollectionView *collectionView)
+static void applyChangesToCollectionView(UICollectionView *collectionView, CKComponentDataSourceAttachController *attachController, CKTransactionalComponentDataSourceState *currentState, CKTransactionalComponentDataSourceAppliedChanges *changes)
```

Comment:
I changed the order of the arguments to more closely match `-transactionalComponentDataSource:didModifyPreviousState:byApplyingChanges:`